tests/functional/test_functional.py

```diff
@@ -136,6 +136,7 @@ def test_mlp_recomputation(gated, activation):
 
 # Takes ~6s, much more if it needs to compile, reducing the hidden size doesn't help.
 @pytest.mark.slow
+@pytest.mark.skip("Dropless MoE is broken")
```

Comment:
btw, torch now comes with grouped_mm, which is much faster than naive looping... https://github.com/huggingface/transformers/pull/42697